mungegithub/mungers/close-stale.go

```diff
@@ -109,6 +99,20 @@ func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool)
 	return lastHuman, true
 }
 
+func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	pr, ok := obj.GetPR()
+	if !ok {
+		return nil, ok
+	}
+
+	comments, ok := obj.ListReviewComments()
+	if !ok {
+		return nil, ok
+	}
+
+	return findLastHumanPullRequestUpdateHelper(pr.CreatedAt, comments)
```

Comment:
Yes, and you can use the pinger to post the comment only once, and find out if it's time to repost.

It can be used to make sure you post a message only once. For example:
```
lastHumanComment := c.LastComment(comments, c.HumanActor(), prCreatedAt)
pinger := NewPinger("INACTIVE-REVIEWER").SetMaxCount(1).SetTimePeriod(time.Hour * 7 * 24)
if !pinger.IsMaxReached(coments, lastHumanComment) { // Verifies if we already added the reviewer/posted the comment
    assign(....)
    notif := pinger.PostNotification(comments, "Some Description", lastHumanComment)
    if notif != nil {
        obj.WriteCommand(notif.String())
    }
}
```

Disclaimer: Written in the github comment box directly, YMMV